shared/middleware/animation_library_api.rb

```diff
@@ -19,7 +19,7 @@ class AnimationLibraryApi < Sinatra::Base
   #
   # Retrieve a file from the animation library
   #
-  get %r{/api/v1/animation-library/([^/]+)/([^/]+)} do |version_id, animation_name|
+  get %r{/api/v1/animation-library/([^/]+)/(.+)} do |version_id, animation_name|
```

Comment:
This allows the "download an animation" endpoint to include slashes in the animation name.